reana-ui/src/components/TopHeader.js

```diff
@@ -2,66 +2,61 @@
 	-*- coding: utf-8 -*-
 
 	This file is part of REANA.
-	Copyright (C) 2018 CERN.
+	Copyright (C) 2020 CERN.
 
   REANA is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 */
 
 import React from "react";
-import { Segment, Image, Menu, Icon } from "semantic-ui-react";
+import { Image, Icon, Popup, List } from "semantic-ui-react";
 import { useDispatch } from "react-redux";
 import { Link } from "react-router-dom";
 import LogoImg from "../images/logo-reana.svg";
-import Cookies from "universal-cookie";
 import { userLogout } from "../actions";
+import config from "../config";
 
 import styles from "./TopHeader.module.scss";
 
-const cookies = new Cookies();
-
-const REANA_DOCS_URL = "https://reana.readthedocs.io/en/latest/";
-const REANA_SITE_URL = "http://www.reana.io";
-
 export default function TopHeader() {
   const dispatch = useDispatch();
-
-  /**
-   * Logs out the current session
-   */
   const logOut = () => {
-    cookies.remove("user_token");
-    cookies.remove("jwt_token");
-    cookies.remove("workflow-id");
-    cookies.remove("workflow-name");
-    cookies.remove("workflow-run");
-    cookies.remove("workflow-created");
-    cookies.remove("workflow-status");
     dispatch(userLogout());
   };
 
   return (
-    <Segment secondary clearing attached="top" padded>
+    <div className={styles["top-header"]}>
       <Link to="/">
-        <Image
-          src={LogoImg}
-          size="small"
-          floated="left"
-          style={{ margin: "0px" }}
-        />
+        <Image src={LogoImg} size="small" />
       </Link>
-      <Menu size="large" floated="right">
-        <Menu.Item href={REANA_SITE_URL} target="_blank">
-          About
-        </Menu.Item>
-        <Menu.Item href={REANA_DOCS_URL} target="_blank">
-          Documentation
-        </Menu.Item>
-        <Menu.Item onClick={logOut} className={styles["logout-button"]}>
-          <Icon inverted name="user" />
-          Log out
-        </Menu.Item>
-      </Menu>
-    </Segment>
+      <section>
+        <a href={config.docs_url} target="_blank" rel="noopener noreferrer">
+          <Icon name="book"></Icon> Help
```

Comment:
The only thing I don't like is that this icon is the same as the docs one on the footer, could we use something else?

Making reference to the community, icon `group`:
<img width="1913" alt="Screenshot 2020-01-31 at 13 46 59" src="https://user-images.githubusercontent.com/9035606/73540467-51388900-4430-11ea-9c0e-787d3710dfc1.png">
Classic question mark, icon `question circle outline`:
<img width="1909" alt="Screenshot 2020-01-31 at 13 45 41" src="https://user-images.githubusercontent.com/9035606/73540468-51388900-4430-11ea-9dc9-ba0e73bced31.png">

Plus, the link to help, should it rather go to the forum instead of the docs? I would expect a place where to (1) check if someone had the same idea/problem and if not (2) ask, WDYT?